src/command-configs/renderHelpPage.ts

```diff
@@ -16,10 +15,23 @@ export function renderHelpPage(params: {
   const { commandConfigs, scriptsRevision, headBranch, config } = params;
 
   const repoLink = new URL(path.join(config.pipelineScripts.repository, "tree", headBranch)).toString();
-  const commandStart = botPullRequestCommentMention;
+  const commandStart = config.botPullRequestCommentMention;
 
   const preparedConfigs = prepareConfigs(commandConfigs);
 
+  // getting list of possible repos, to be able to filter out relevant commands
+  let repos: string[] = Object.values(preparedConfigs).reduce((acc: string[], cmdConfig: CmdJson) => {
+    const currentRepos: string[] = cmdConfig.command.presets
+      ? Object.values(cmdConfig.command.presets)
+          .map((p) => p.repos?.filter((r) => !!r) || [])
+          .flat()
+      : [];
+    return [...acc, ...currentRepos];
+  }, []);
+
+  // deduplicate repos
+  repos = [...new Set(repos)];
```

Comment:
isn't it basically this?
```ts
const reposSet = new Set<String>();

for (const cmdConfig of Object.values(preparedConfigs)) {
  for (const preset of Object.values(cmdConfig.command.presets ?? {}) {
    for (const repo of preset.repos ?? []) {
      reposSet.add(repo);
    }
  }
}

const repos = [...reposSet];
```

Reply:
yup, thanks man, it's simpler will update it

Reply:
updated [75ba8bb](https://github.com/paritytech/command-bot/pull/208/commits/75ba8bb8dbaea93f5fe12b70a120cb334754ea7f)